backend/driver.pm

```diff
@@ -45,6 +47,12 @@ sub new {
     $self->{backend}      = "backend::$name"->new();
     $self->{backend_name} = $name;
 
+    session->on(
+        collected_orphan => sub {
+            my ($session, $p) = @_;
+            printf STDERR "Driver backend collected unknown process with pid " . $p->pid . " and exit status: " . $p->exit_status . "\n";
+        });
+
```

Comment:
This seems to be a common pattern.

Reply:
Yup, but it is also in os-autoinst/openQA as well